plop-templates/components/{{kebabCase name}}/stories/{{kebabCase name}}.stories.js

```diff
@@ -1,9 +1,29 @@
-import { storiesOf } from '@storybook/html';
-import '@tradeshift/elements';
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import { withKnobs, text, select, boolean } from '@storybook/addon-knobs';
+
+import { helpers } from '@tradeshift/elements';
 import '@tradeshift/elements.{{kebabCase name}}';
 
+// import { sizes } from '../src/utils';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACommentedCode)

Reply:
here